generators/tests/org/jetbrains/kotlin/generators/tests/GenerateTests.kt

```diff
@@ -171,15 +169,45 @@ fun main(args: Array<String>) {
 
         testGroup("plugins/jvm-abi-gen/test", "plugins/jvm-abi-gen/testData") {
             testClass<AbstractCompareJvmAbiTest> {
-                model("compare", recursive = false, extension = null)
+                model("compare", recursive = false, extension = null, targetBackend = TargetBackend.JVM)
             }
 
             testClass<AbstractJvmAbiContentTest> {
-                model("content", recursive = false, extension = null)
+                model("content", recursive = false, extension = null, targetBackend = TargetBackend.JVM)
             }
 
             testClass<AbstractCompileAgainstJvmAbiTest> {
-                model("compile", recursive = false, extension = null)
+                model("compile", recursive = false, extension = null, targetBackend = TargetBackend.JVM)
+            }
+
+            testClass<AbstractIrCompareJvmAbiTest> {
+                model("compare", recursive = false, extension = null, targetBackend = TargetBackend.JVM_IR)
+            }
+
+            testClass<AbstractIrJvmAbiContentTest> {
+                model("content", recursive = false, extension = null, targetBackend = TargetBackend.JVM_IR)
+            }
+
+            testClass<AbstractIrCompileAgainstJvmAbiTest> {
+                model("compile", recursive = false, extension = null, targetBackend = TargetBackend.JVM_IR)
+            }
+        }
+
+        testGroup(
```

Comment:
Why these tests are not targeted to IR?

Reply:
Because the old implementation of jvm-abi-gen does not really support the IR backend. It is always running the old backend as part of `JvmAbiAnalysisHandlerExtension` ([here](https://github.com/JetBrains/kotlin/blob/a54e758189c2048c5d5550936fad3d8afa5c1703/plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiAnalysisHandlerExtension.kt#L54)).

...and this just reminded me that the tests need to pass `useOldBackend` instead of `useIR` now... that's also fixed.